eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -232,16 +339,16 @@ function Login() {
     Write-Verbose "Logging in with provisioner"
     $provisionerSecret = ConvertTo-SecureString -String $ProvisionerApplicationSecret -AsPlainText -Force
     $provisionerCredential = [System.Management.Automation.PSCredential]::new($ProvisionerApplicationId, $provisionerSecret)
-    Connect-AzAccount -Force -Tenant $TenantId -Credential $provisionerCredential -ServicePrincipal -Environment $Environment
-    Select-AzSubscription -Subscription $SubscriptionId -Confirm:$false
+    Connect-AzAccount -Force -Tenant $TenantId -Credential $provisionerCredential -ServicePrincipal -Environment $Environment -WhatIf:$false
+    Select-AzSubscription -Subscription $SubscriptionId -Confirm:$false -WhatIf:$false
```

Comment:
Why not just not pass them? At least then a `$WhatIfPreference` override would be possible.

Reply:
My intention here with `WhatIf` mode is that you see what delete operations the script will take, without taking them. If we use the default `WhatIf` everywhere, then the script just fails to log in and you can't get an accurate estimate of what the behavior will be.